src/mono/wasm/runtime/strings.ts

```diff
@@ -276,3 +276,13 @@ export function js_string_to_mono_string_new(string: string): MonoString {
         temp.release();
     }
 }
+
+export function get_hash_code(string: string) {
```

Comment:
this probably needs a comment explaining the existence of a special purpose hashing routine when we could use the hashes provided by the browser or the managed runtime instead

Reply:
personally, i would prefer to use a strong hash since a hash collision could result in loading the wrong memory

Reply:
ok, I will use `crypto` API